massa-models/src/slot.rs

```diff
@@ -6,8 +6,8 @@ use super::{
     },
     with_serialization_context,
 };
-use crate::constants::SLOT_KEY_SIZE;
-use crate::error::ModelsError;
+use crate::{constants::SLOT_KEY_SIZE, node_configuration::THREAD_COUNT};
```

Comment:
does this interfere with the tests when we use a different number of threads for a specific test ?

Reply:
I have no idea, it needs to be verified.